Security/Guidelines/OpenSSH.mediawiki

```diff
@@ -262,7 +262,7 @@ $ ssh-add ~/.ssh/id_..._mozilla... # <= replace by your key's path
 
 <source>
 # Add configuration to ~/.ssh/config
-host *.mozilla.com
+host *.mozilla.com !ssh.mozilla.com
```

Comment:
I would feel a lot better about this exclusion (who knew openssh could even do that!) if you included a #comment here explaining why the exclusion is necessary.